_workinggroups/toolsandpackaging.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Software Developer Tools and Packaging
+layout: plain
+redirect_from:
+  - /workinggroups/2015/11/04/packaging.html
+  - /activities/packaging.html
+  - /workinggroups/packaging.html
+  - /workinggroups/softwaredevelopertools.html
+---
+
+## Software Developer Tools
+
+The Software Developer Tools Working Group aims to help the HEP community in developing the latest, modern feature-rich projects
+with the aid of software tools such as code editors, static/dynamic code analyzers, compilers, debuggers, performance analyzers and so on.
```

Comment:
Sorry for being a pain: did we note decide at some point to use UK English, hence `analysers`, etc.?

Reply:
I think that was for the CWP, but, indeed, no harm in being consistent!